src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Section.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public class Section
+    {
+        /// <summary>
+        /// Section Title
+        /// </summary>
+        public string Title;
+
+        /// <summary>
+        /// Response rendered under this section
+        /// </summary>
+        public Response Data;
+
+        /// <summary>
+        /// Whether section is expaned at beginning
+        /// </summary>
+        public bool IsExpanded;
+
+        /// <summary>
+        /// create an instance of Section class
+        /// </summary>
+        /// <param name="data"> </param>
+        /// <param name="title">Section Title</param>
+        /// <param name="isExpanded">Default expand state of section</param>
+        public Section(Response data, string title = "", bool isExpanded = true)
```

Comment:
nitpick - the logical order of the Section is the title first and then inner response. Can you update the order of parameters in the ctor() as well.

Reply:
Updated